src/hiero_sdk_python/tokens/token_key_validation.py

```
@@ -5,22 +5,27 @@
+
 from enum import Enum
 from typing import Any
 from hiero_sdk_python.hapi.services import basic_types_pb2
 
+
 class TokenKeyValidation(Enum):
     """
     Enum for token key validation modes:
 
-      • FULL_VALIDATION – perform all validation checks  
+      • FULL_VALIDATION – perform all validation checks
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)